packages/just-scripts/src/utils/index.ts

```diff
@@ -1,7 +1,4 @@
 export * from './exec';
-export * from './interfaces/PackageJson';
-export * from './interfaces/RushJson';
-export * from './logger';
```

Comment:
- PackageJson was moved to a different location, not under the utils folder
- RushJson types are removed: they weren't used anywhere, and weren't re-exported from `just-scripts`
- This was a re-export of the logger package which is now redundant